src/test/java/com/epam/lab/paymentsystem/util/DbTestUtil.java

```diff
@@ -0,0 +1,33 @@
+package com.epam.lab.paymentsystem.util;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+import javax.sql.DataSource;
+import org.springframework.context.ApplicationContext;
+import org.springframework.core.env.Environment;
+
+public final class DbTestUtil {
+
+  private DbTestUtil() {
+  }
+
+  public static void resetAutoIncrementColumns(ApplicationContext applicationContext,
```

Comment:
Are you sure you want to do it this way? Can't you just use smth like that:
```
public int testUserId = 1;
...
public void init() {
   ...
   User user = new User("test", "test", "test", role);
   user.setId(testUserId);
   user = userRepository.save(user);
   ...
}
```

Reply:
No, autogeneration defined in our database, so this approach(setting id in entity) does not affect the ID column (I have already tried)